jerry-core/ecma/base/ecma-helpers-conversion.c

```diff
@@ -917,227 +919,512 @@ ecma_number_to_int32 (ecma_number_t num) /**< ecma-number */
 } /* ecma_number_to_int32 */
 
 #if CONFIG_ECMA_NUMBER_TYPE == CONFIG_ECMA_NUMBER_FLOAT64
+
```

Comment:
Same as before this patch. :) I didn't touched that part. There is another algorithm for float32 to string conversion.